lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3577,7 +3587,9 @@ namespace {
         if (auto structResult = dyn_cast<StructDecl>(result))
           structResult->setIsCxxNonTrivial(
               !cxxRecordDecl->isTriviallyCopyable());
-
+        for (auto &getterAndSetter : Impl.GetterSetterMap){
+            result->addMember(get<0>(getterAndSetter.second));
```

Comment:
If we can't get rid of the tuple, please add a comment explaining that this is the vardecl/property itself.